runtime/doc/news.txt

```diff
@@ -39,6 +39,11 @@ TREESITTER
 
 • todo
 
+UI
+
+• `progress` attribute removed form |ui-messages| msg_show event
```

Comment:
Unnecessary for such a short-lived feature on master.

Reply:
True. But shouldn't we still do it to follow procedure :] ?